ReSwift.xcodeproj/project.pbxproj

```diff
@@ -547,11 +547,11 @@
 					};
 					25DBCF7A1C30C4AA00D63A58 = {
 						CreatedOnToolsVersion = 7.2;
-						LastSwiftMigration = 0900;
+						LastSwiftMigration = "";
```

Comment:
could we remove these lines completely?

Reply:
I think we can somewhat remove them... I've now removed them and then ran a clean & build on each scheme, then resolved any warnings, and I also removed the target specifiers for swift version, so everything falls back to the project setting.